src/cgame/default/views/PlayerModelView.c

```diff
@@ -150,16 +150,22 @@ static void render(View *self, Renderer *renderer) {
 
 		const vec_t xmin = ymin * aspect;
 		const vec_t xmax = ymax * aspect;
+		matrix4x4_t mat;
 
-		glFrustum(xmin, xmax, ymin, ymax, NEAR_Z, FAR_Z);
+		Matrix4x4_FromFrustum(&mat, xmin, xmax, ymin, ymax, NEAR_Z, FAR_Z);
+
+		glLoadMatrixf((GLfloat *) mat.m);
 
 		glMatrixMode(GL_MODELVIEW);
-		glLoadIdentity();
+
+		Matrix4x4_CreateIdentity(&mat);
 
 		// Quake is retarded: rotate so that Z is up
-		glRotatef(-90.0, 1.0, 0.0, 0.0);
-		glRotatef( 90.0, 0.0, 0.0, 1.0);
-		glTranslatef(64.0, 0.0, -8.0);
+		Matrix4x4_ConcatRotate(&mat, -90.0, 1.0, 0.0, 0.0);
+		Matrix4x4_ConcatRotate(&mat,  90.0, 0.0, 0.0, 1.0);
+		Matrix4x4_ConcatTranslate(&mat, 64.0, 0.0, -8.0);
+
+		glLoadMatrixf((GLfloat *) mat.m);
 		
 		glPushMatrix();
```

Comment:
It looks like you've added R_Push / PopMatrix to the renderer. We should therefore expose those via cg_export_t, and use `cge.PushMatrix` and `cge.PopMatrix` here.